addons/sale/data/sale_demo.xml

```diff
@@ -548,6 +584,45 @@
             <field name="price_unit">12.50</field>
         </record>
 
+        <record id="sale_order_19" model="sale.order">
+            <field name="partner_id" ref="base.res_partner_3"/>
+            <field name="partner_invoice_id" ref="base.res_partner_address_25"/>
+            <field name="partner_shipping_id" ref="base.res_partner_address_25"/>
+            <field name="user_id" ref="base.user_demo"/>
+            <field name="pricelist_id" ref="product.list0"/>
+            <field name="team_id" ref="sales_team.team_sales_department"/>
+        </record>
+
+        <record id="sale_order_line_42" model="sale.order.line">
+            <field name="order_id" ref="sale_order_19"/>
+            <field name="name">Mouse, Wireless</field>
+            <field name="product_id" ref="product.product_product_12"/>
+            <field name="product_uom_qty">2</field>
+            <field name="product_uom" ref="uom.product_uom_unit"/>
+            <field name="price_unit">12.50</field>
+        </record>
+
+        <record id="sale_order_line_43" model="sale.order.line">
+            <field name="order_id" ref="sale_order_19"/>
+            <field name="name">Section 1 long long long long long long long long long long long long long long long long </field>
+            <field name="line_type">line_section</field>
+        </record>
+
+        <record id="sale_order_line_44" model="sale.order.line">
+            <field name="order_id" ref="sale_order_19"/>
+            <field name="name">Mouse, Wireless</field>
+            <field name="product_id" ref="product.product_product_12"/>
+            <field name="product_uom_qty">2</field>
+            <field name="product_uom" ref="uom.product_uom_unit"/>
+            <field name="price_unit">12.50</field>
+        </record>
+
+        <record id="sale_order_line_45" model="sale.order.line">
+            <field name="order_id" ref="sale_order_19"/>
+            <field name="name">Note note note note note</field>
```

Comment:
Same here. It should be something more functional than 'prout prout' or 'Lorem Ipsum' :p